doc/source/index.rst

```diff
@@ -75,7 +75,9 @@ Here is a brief example of how you use PyAnsys Math:
 
     print(w)
 
-.. code:: output
+.. rst-class:: sphx-glr-script-out
```

Comment:
Why this?

Reply:
This is to render like this: 
![image](https://user-images.githubusercontent.com/78221213/231387281-35853193-7a1f-4091-a364-2b7b85163e83.png)

Reply:
Very well then!